packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -27,53 +28,47 @@ export function SquadHeaderBar({
   className,
   ...props
 }: SquadHeaderBarProps): ReactElement {
-  const { user } = useAuthContext();
+  const [, copying, copyLink] = useCopySquadInvitation({ squad });
   const { onMenuClick } = useContextMenu({ id: 'squad-menu-context' });
-  const { openModal } = useLazyModal();
-  const openSquadInviteModal = () =>
-    openModal({
-      type: LazyModal.SquadInvite,
-      props: {
-        initialSquad: squad,
-      },
-    });
+  const { sidebarRendered } = useSidebarRendered();
 
   return (
     <div
       {...props}
-      className={classNames(
-        'flex laptop:flex-row flex-col justify-center items-center gap-4 w-full',
-        className,
-      )}
+      className={classNames('flex flex-row gap-4 h-fit', className)}
     >
-      <SquadMemberShortList
-        squad={squad}
-        members={members}
-        memberCount={memberCount}
-      />
-      <span className="flex flex-row gap-4">
-        <Button className="btn-secondary" onClick={openSquadInviteModal}>
-          Invite
-        </Button>
-        <SimpleTooltip placement="top" content="Feedback">
-          <Button
-            tag="a"
-            target="_blank"
-            rel="noopener noreferrer"
-            href={`${squadFeedback}#user_id=${user.id}&squad_id=${squad.id}`}
-            className="btn-secondary"
-            icon={<FeedbackIcon aria-label="squad-feedback-icon" />}
-          />
-        </SimpleTooltip>
-        <SimpleTooltip placement="top" content="Squad options">
-          <Button
-            className="btn-secondary"
-            icon={<MenuIcon />}
-            onClick={onMenuClick}
-            aria-label="Squad options"
-          />
-        </SimpleTooltip>
-      </span>
+      <Button
+        className="btn-primary"
+        onClick={() => copyLink()}
+        icon={<AddUserIcon />}
+        disabled={copying}
+      >
+        Copy invitation link
+      </Button>
+      {sidebarRendered && (
+        <SquadMemberShortList
+          squad={squad}
+          members={members}
+          memberCount={memberCount}
+        />
+      )}
```

Comment:
Big request maybe, but is there anyway we can solve the duplicate component with CSS? (order maybe?)

Reply:
I tried the order css but failed miserably. I have a feeling it should be possible too but I have already spent lots of minutes on it and thought of doing some time when we get wiggle room.